hapi-fhir-jpaserver-subscription/pom.xml

```diff
@@ -20,11 +20,11 @@
 			<artifactId>hapi-fhir-server</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-		<dependency>
+		<!--<dependency>
```

Comment:
why commented out?

if its really necessary for now, add TODO to revisit later.